sdks/python/setup.py

```diff
@@ -131,16 +131,14 @@ def get_version():
     # is Python standard since Python 3.7 and each Python version is compatible
     # with a specific dataclasses version.
     'dataclasses;python_version<"3.7"',
-    # orjson, only available on Python 3.6 and above
-    'orjson<4.0;python_version>="3.6"',
+    'orjson<4.0',
```

Comment:
I believe we still support Python 3.5, so we may not be able to make this change?

Reply:
`python_requires` below shows `>=3.6`: https://github.com/apache/beam/blob/7497a2c4662a826b1e6a809bba3fbaf52f02fac7/sdks/python/setup.py#L255

Reply:
ah great. Thanks!